backend/src/routers/news.py

```diff
@@ -116,21 +108,11 @@ async def news_summary(
         payload: NewsSumaryRequestSchema, user=Depends(authenticate_user_token)
 ):
     summary_response = {}
-    summary_generation_messages = [
-        {
-            "role": "system",
-            "content": "你是一個新聞摘要生成機器人，請統整新聞中提及的影響及主要原因 (影響、原因各50個字，請以json格式回答 {'影響': '...', '原因': '...'})",
-        },
-        {"role": "user", "content": f"{payload.content}"},
-    ]
+    
+    llm_client = LLMClient(api_key="xxx")
```

Comment:
Again, I don't think one should put the api key this way.
env aside, the key should as least be in a var.